project 3/CODE/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand|Ubuntu:700" rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+
+    <div id="hour6am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour6am"><img src="images/6am.jpg" alt=''></a>
+        <h2>6:00 AM</h2>
+        <p>Here is a constant debate of whether to get up or sleep for another 15 minutes and then get up. This is
+            really tough for me.</p>
+    </div>
+    <div id="hour7am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour7am"><img src="images/7am.jpg" alt=''></a>
+        <h2>7:00 AM</h2>
+        <p>At this point I have decided to get up and be a functioning member of society, I just need my body to know
+            that. Come on.</p>
+    </div>
+    <div id="hour8am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour8am"><img src="images/8am.jpg" alt=''></a>
+        <h2>8:00 AM</h2>
+        <p> I am on my way to class this is the bliss before I realized that I had forgotten my hard drive. Which for
+            Tuesdays for me is super important because today I tend to stare at my screen for more than twelve or
+            fourteen hours a day, all while soley writing code. Either for Dae In Chung creating interactive things for
+            Gen Type or for you, Angelique, creating this timeline while my brain melts out of my ear.
+
+            Don't get me wrong I love Graphic Design. It just gets to be a lot sometimes. I need a good cry from time to
```

Comment:
Yeah, your Tuesday schedule would make me weepy some days, too. While my typical Tuesday is from 9am - 10:30pm and involves a lot of screens, I'm not always pouring new info into my brain that whole time.